packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.tsx

```diff
@@ -14,24 +14,22 @@ export interface ITeachingBubbleState {
 }
 
 export class TeachingBubble extends BaseComponent<ITeachingBubbleProps, ITeachingBubbleState> {
-
-  // Specify default props values
-  public static defaultProps = {
```

Comment:
Unfortunately this should be deprecated and marked for removal in a future release. We want to avoid changing public contracts, even if they were added erroneously.

Reply:
I approved but fix this first and then finish your pr 👍

Reply:
Does this new commit satisfy that req?

Reply:
Yes :)